src/main/kotlin/uk/gov/justice/digital/hmpps/nomisprisonerapi/movements/MovementsResource.kt

```diff
@@ -55,4 +55,42 @@ class MovementsResource(
   fun getTemporaryAbsencesAndMovements(
     @Schema(description = "Offender number (NOMS ID)", example = "A1234BC") @PathVariable offenderNo: String,
   ): OffenderTemporaryAbsencesResponse = movementsService.getTemporaryAbsencesAndMovements(offenderNo)
+
+  @PreAuthorize("hasRole('ROLE_NOMIS_MOVEMENTS')")
+  @GetMapping("/movements/{offenderNo}/temporary-absences/application/{applicationId}")
+  @Operation(
+    summary = "Get a specific temporary absence application for an offender",
```

Comment:
The main change is a new endpoint to get a movement application to sync it to DPS.